docs/src/keps/index.rst

```diff
@@ -0,0 +1,13 @@
+Klio Enhancement Proposals
+==========================
+
+These pages provide an overview of development decisions and priorities for Klio.
+Specifically, it contains Klio Enhancement Proposals (KEPs) – suggested changes to the ecosystem and to processes – in various stages of discussion or completion (see :doc:`KEP 0 <kep-001>`).
```

Comment:
Is this supposed to be `KEP 001 <kep-001>`

Reply:
The title of the KEP is "KEP 0" (like [PEP 1](https://www.python.org/dev/peps/pep-0001/) and [NEP 0](https://numpy.org/neps/nep-0000.html)). But it's easier to name docs with the preceding 0s because then they keep their order. I can update the language to reflect how titling should be.

Reply:
Then wouldn't this be `KEP 1` and not `KEP 0`?